.reuse/dep5

```diff
@@ -0,0 +1,38 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```

Comment:
Does this essentially duplicate `debian/copyright`? Is there any way to use one as the other like a symlink? Not critical, but this seems like the type of information that would only get updated in one place or the other but not both.

Reply:
tl;dr: Agreed, it’s awkward to have two sort-of-duplicate files, but I can’t think of a better approach. Suggestions welcome!

---

It is the same format as `debian/copyright`, but I’m not sure they can be merged as they have slightly conflicting purposes. `debian/copyright` aims to give the copyright/licensing information for every file in the repository, whereas `.reuse/dep5` is a way to give copyright/licensing information for files where it can’t be included inline. Its use is discouraged in favour of putting the licensing/copyright inside the files themselves.

If `.reuse/dep5` were to be merged/linked with `debian/copyright`, it would then contain licensing/copyright information for every file in the repository, which would go against the intent of using it only for difficult files.

One [alternative to using `.reuse/dep5`](https://reuse.software/spec/#comment-headers) is to instead add a `.license` file for each file listed in it (so `tests/gpghome/keyid.license` to accompany `tests/gpghome/keyid`, etc.). I shied away from that as it would pollute the repository a bit (and wildcards can’t be used, so we’d need O(N) files for `debian/**` rather than one entry in `.reuse/dep5`). I can give that a go if you think it’d be better.

Reply:
I think this kind of duplication is better than littering the source tree with `.license` files.